core/deployment/src/main/java/io/quarkus/deployment/steps/NativeImageConfigBuildStep.java

```diff
@@ -77,11 +78,18 @@ void build(SslContextConfigurationRecorder sslContextConfigurationRecorder,
         Boolean sslNativeEnabled = isSslNativeEnabled(sslNativeConfig, extensionSslNativeSupport);
         if (sslNativeEnabled) {
             // This makes the native image dependent on the local path used to build it.
+            // This is useful for testing but the user will have to override it.
             String graalVmHome = System.getenv("GRAALVM_HOME");
             if (graalVmHome != null) {
-                Path graalVmCacertsPath = Paths.get(graalVmHome, "jre", "lib", "security", "cacerts");
-                // This is useful for testing but the user will have to override it.
-                sslTrustStoreSystemProperty.produce(new SslTrustStoreSystemPropertyBuildItem(graalVmCacertsPath.toString()));
+                Path jdk8GraalVmCacertsPath = Paths.get(graalVmHome, "jre", "lib", "security", "cacerts");
+                if (Files.exists(jdk8GraalVmCacertsPath)) {
+                    sslTrustStoreSystemProperty
+                            .produce(new SslTrustStoreSystemPropertyBuildItem(jdk8GraalVmCacertsPath.toString()));
+                } else {
+                    sslTrustStoreSystemProperty.produce(
+                            new SslTrustStoreSystemPropertyBuildItem(
+                                    Paths.get(graalVmHome, "lib", "security", "cacerts").toString()));
+                }
```

Comment:
Since the only thing that changes is the `Path`, I'd just change that: 

```suggestion
                Path graalVmCacertsPath = Paths.get(graalVmHome, "jre", "lib", "security", "cacerts");
                if (!Files.exists(graalVmCacertsPath)) {
                   graalVmCacertsPath = Paths.get(graalVmHome, "lib", "security", "cacerts");
                }
                // This is useful for testing but the user will have to override it.
                sslTrustStoreSystemProperty.produce(new SslTrustStoreSystemPropertyBuildItem(graalVmCacertsPath.toString()));
                
```

Reply:
Next time I will just take my day off rather than trying to work :)

Done, thanks!